values.yaml

```diff
@@ -53,6 +53,13 @@ kpi:
     minAvailable: 1
   env:
     normal:
+      # By default, the following are set but can be overridden
+      DEFAULT_FROM_EMAIL: "support@kobotoolbox.org"
+      ENKETO_EXPRESS_PUBLIC_SUBDOMAIN: "ee"
+      KOBOCAT_PUBLIC_SUBDOMAIN: "kc"
+      KOBOFORM_PUBLIC_SUBDOMAIN: "kf"
```

Comment:
I'm pretty indifferent, but `kobo` as suggested in your PR description might be a better default. Who knows: maybe the future single domain for everything will be totally different, like `app`

Reply:
I'll go with kobo.

Reply:
Fun fact, I would have said. let's keep `kf`. It's short, it's the one used by kobo-install and it matches our production (and probably many self hosters).